pybytes/README.md

```diff
@@ -4,7 +4,20 @@
 
 Please note that these examples only work when using the `pybytes` firmware.
 
-# Pytrack
+# Pytrack, Pysense, Pyscan
 - To run pytrack on individuals module user will need to create one pybytes_config.json file on root which usually contains data related to device e.g Network preferance, device id, server, username, password etc.
 
 - User needs to upload all the files to module in order to get data on pybytes dashboard.
+
+Note: For using pyscan user needs to upload either MFRC630.mpy or MFRC630.py.
+MFRC630.mpy if he is using WiPy else MFRC630.py will work.
```

Comment:
We always recommend using the `.mpy` version, it's smaller so the user has more space for their code.

Reply:
Added to readme file